app/controllers/portal/tax_returns_controller.rb

```diff
@@ -42,7 +42,7 @@ def show; end
     private
 
     def load_client_tax_return
-      @tax_return = current_client.tax_returns.includes(client: [:intake]).find_by(id: params[:tax_return_id])
```

Comment:
why was this removed in this case? Is it b/c this `success` page (which is the only page that utilizes `load_client_tax_return` method does not need client/intake records